ipa-server/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,61 @@
+---
+namespace: ipa
+app_name: ipa
+base_openshift_url: example.com 
+realm: "EXAMPLE.COM"
+
+params:
+  IPA_SERVER_SERVICE: "{{ app_name }}"
+  IPA_SERVER_IMAGE: "freeipa-server:rhel-7"
+  IPA_SERVER_HOSTNAME: "{{ app_name }}.apps.{{ base_openshift_url }}"
+  IPA_SERVER_INSTALL_OPTS: "-U -r {{ realm }} --setup-dns --no-forwarders --no-ntp"
```

Comment:
Curious to know what the `--setup-dns` will do in this case. We normally host DNS externally to the OpenShift cluster, so having another DNS server running in here may potentially mean "out-of-sync" DNS ... The good news is that IdM may be the only one confused. Did you try without the `--setup-dns` flag? I suspect that the "install" may fail if it's not specified?

Reply:
I'll need to test this out. My initial experience with removing some of these didn't go well, but I also tried pulling a few out at a time. Let me try just removing that flag and I'll circle back.

Reply:
ok - that could be done post-merge as well. Just curious if this was tried and what the outcome was.

Reply:
@oybed I dug into this a little bit and end up with the same result (not good). May be best left for a future iteration of digging into the `ipa-server-install` command flags.